lib/taste_tester/tunnel.rb

```diff
@@ -44,15 +44,13 @@ def run
     end
 
     def cmd
-      if TasteTester::Config.user != 'root'
-        pid = '$$'
-      else
-        pid = '\\$\\$'
```

Comment:
Anyway maybe something else changed? You should check in with @mikedodge04 or @natewalck  - they use TT via sudo, so if they can confirm this doesn't break them, I'm happy with it.